main/store/actions/index.js

```diff
@@ -567,6 +568,11 @@ module.exports = {
       return balances
     })
   },
+  removeBalancesBySet: (u, address, tokensToRemove) => {
```

Comment:
this is going back to the first way where it will trigger a re-render for every account no matter if the balances are removed or not

Reply:
also I think we can just call it `removeBalances`, the fact that the last parameter is a `Set` is either inferred or would be explicit if this file was using TS

Reply:
We perform a check prior to calling the action if its needed

I added 'BySet' because there was already a 'removeKnownTokens' action and thought it best to be consistent as they share some logic!

Reply:
ahh I see you're only calling this when needed, got it. don't `removeKnownTokens` and `removeKnownTokensBySet` basically do the same thing?

Reply:
Yup, I can also refactor current uses of removeKnownTokens to use a set 👍